examples/classic_controllers/classic_controllers_synch_motor_manual_example.py

```diff
@@ -0,0 +1,113 @@
+from classic_controllers import Controller
+from externally_referenced_state_plot import ExternallyReferencedStatePlot
+import gym_electric_motor as gem
+from gym_electric_motor.visualization import MotorDashboard
+import numpy as np
+
+if __name__ == '__main__':
+
+    """
+        motor type:     'PMSM'      Permanent Magnet Synchronous Motor
+                        'SynRm'     Synchronous Reluctance Motor
+                        
+        control type:   'S'         Speed Control
+                        'T'         Torque Control
+                        'C'         Current Control
+
+        action_type:    'AbcCont'   Continuous Action Space in ABC-Coordinates
+                        'Finite'    Discrete Action Space
+    """
+
+    motor_type = 'PMSM'
```

Comment:
All SynRm variants resulted in an 'no registered env' type of error.

Reply:
@fbook98 Let's have a cast to lower on all string user inputs with ```s.lower()``` and compare with the corresponding expected lowercase strings

Reply:
@wkirgsn, I dont think that we can do anything here (at least regarding the environment ids). This would require changes in the registration and creation of environments within code of OpenAI-GYM.